suites/apis/metadataIngestionTest.js

```diff
@@ -6,7 +6,7 @@
     within the sower config block (ingest-metadata-manifest & get-dbgap-metadata)
  3. Metadata Service (mds) must also be deployed
 */
-Feature('Metadata Ingestion');
+Feature('Metadata Ingestion @requires-sower');
```

Comment:
does this also require mds / indexd / fence?

Reply:
This is an initial WIP PR. We can keep adding stuff to this